app/src/main/java/com/teamagam/gimelgimel/app/message/viewModel/adapter/MessageViewHolderBindVisitor.java

```diff
@@ -88,44 +104,44 @@ private void initViewHolder() {
     setGeoPanelVisibility(View.GONE);
   }
 
-  private void setMessageDetails(MessageApp message) {
+  private void setMessageDetails(Message message) {
     setSenderName(message);
     setDate(message);
   }
 
-  private void setSenderName(MessageApp message) {
+  private void setSenderName(Message message) {
```

Comment:
It was like that before you changed the code, but I think you can change the setter methods signature to get the data itself (e.g. `setSenderName(String senderName)`), instead of getting the message.